funcs_scripts.php

```diff
@@ -150,6 +150,11 @@ function is_module()
         return false;
     }
 
+    if (is_gha_repository()) {
+        // gha-generate-matrix has a composer.json, but it's not a module
```

Comment:
```suggestion
```
This condition speaks for itself - gha repositories aren't modules.

Reply:
Updated